app/controllers/admin/posts_controller.rb

```diff
@@ -1,5 +1,6 @@
-class PostsController < ApplicationController
+class Admin::PostsController < ApplicationController
   before_action :set_post, only: [:show, :edit, :update, :destroy]
+  layout 'admin'
```

Comment:
Why don't you introduce `Admin::ApplicationController` to DRY `layout` declaration?

```ruby
class Admin::ApplicationController < ActionController::Base
  layout 'admin'
end

class Admin::PostsController < Admin::ApplicationController
end
```